projects/plugins/jetpack/extensions/blocks/conversation/edit.js

```diff
@@ -42,7 +44,7 @@ function ConversationEdit( { className, attributes, setAttributes } ) {
 		updatedParticipant =>
 			setAttributes( {
 				participants: participants.map( participant => {
-					if ( participant.participantSlug !== updatedParticipant.participantSlug ) {
+					if ( participant.slug !== updatedParticipant.slug ) {
```

Comment:
Most of the changes in this file are simply updating from `participant.participantSlug` to `participant.slug`